### PR TITLE
use umd, update docs

### PR DESCRIPTION
**Pull request by bcherny**
This PR:
- Wraps each Slick module in a UMD
- Adds a compatibility build for consumers that expected `Slick.Data.DataView` (etc.) as opposed to just `DataView`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,8 +17,8 @@
   ],
   "main": "./slick.grid.js",
   "scripts": {
-    "build": "rollup -c",
-    "watch": "rollup -c --watch",
+    "build": "rollup -c rollup.main.js && rollup -c rollup.compat.js",
+    "watch": "rollup -c rollup.main.js --watch & rollup -c rollup.compat.js",
```
Comment (ghost) [inline package.json:21]: should rollup.compat.js have watch as well?

**Comment by albertywu** [comment]
:+1:

File: tsconfig.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     "preserveConstEnums": true,
     "pretty": true,
     "rootDir": "./src",
-    "target": "es6"
+    "target": "es5"
```
Comment (ghost) [inline tsconfig.json:14]: ur such a good open source citizen

**Review by ghost — APPROVED** [review]
beautiful!

File: package.json (hunk 1 above)
Comment (bcherny) [reply package.json:21]: It does - try re-reading that line

**Pull request merged**